src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -1,83 +1,122 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+import { compose } from "@utilities/compose";
+import { debounce } from "@utilities/debounce";
+import { useDebounce } from "@utilities/useDebounce";
 import { useMachine } from "@xstate/react";
-import { Editor, EditorState, getDefaultKeyBinding, RawDraftContentState, RichUtils } from "draft-js";
-import "draft-js/dist/Draft.css";
-import React, { FC, useRef } from "react";
+import React, { FC, useCallback, useEffect, useMemo, useState } from "react";
+import { BaseEditor, createEditor, Descendant } from "slate";
+import { withHistory } from "slate-history";
+import { Editable, ReactEditor, Slate, withReact } from "slate-react";
 import { DoneInvokeEvent, Interpreter } from "xstate";
 import { ToolbarContext } from "./context/toolbar";
-import { decorators } from "./decorators";
+import { useSoftBreak } from "./hooks/useSoftBreak";
+import { withLinks } from "./plugins/withLinks";
+import { withLists } from "./plugins/withLists";
+import { BlockStyleTypes, renderBlockStyles } from "./renderer/renderBlockStyles";
+import { InlineStyles as InlineStyleTypes, renderInlineStyles } from "./renderer/renderInlineStyles";
```

Comment:
You could use `InlineStyles` directly, no need to rename it

Reply:
a3edaadc4aacbf243e199d269a57a49b4c7dae1f